StripeIdentity/StripeIdentityTests/Snapshot/SelfieWarmupViewSnapshotTest.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  SelfieWarmupViewSnapshotTest.swift
+//  StripeIdentity
+//
+//  Created by Chen Cen on 8/15/23.
+//
+
+import iOSSnapshotTestCase
+@_spi(STP) import StripeCore
+@testable import StripeIdentity
+@_spi(STP) import StripeUICore
+
+class SelfieWarmupViewSnapshotTest: FBSnapshotTestCase {
+
+    override func setUp() {
+        super.setUp()
+        //        recordMode = true
```

Comment:
leftover

Reply:
This is a custom pattern we used for snapshot tests incase later something changed and we need to record again, e.g [this one](https://github.com/stripe/stripe-ios/blob/24b03d173b9ee2cdcf08ae6da8a27d9bcd1cc653/StripeIdentity/StripeIdentityTests/Snapshot/PhoneOtpViewControllerSnapshotTest.swift#L18)

Reply:
Yea I was second guessing that I believe you mentioned this before haha, I'll try to remember 😅